src/metrics_server/connection_metrics.ts

```diff
@@ -93,18 +93,15 @@ export function isValidConnectionMetricsReport(
     return false;
   }
 
-  const requiredUserReportFields = ['userId', 'countries', 'bytesTransferred'];
   const MIN_BYTES_TRANSFERRED = 0;
   const MAX_BYTES_TRANSFERRED = 1 * Math.pow(2, 40); // 1 TB.
   for (const userReport of testObject.userReports) {
-    // Test that each `userReport` contains the required fields.
-    for (const fieldName of requiredUserReportFields) {
-      if (!userReport[fieldName]) {
-        return false;
-      }
+    // We require at least the userId or the country to be set.
+    if (!userReport.userId && (!userReport.countries || userReport.countries.length === 0)) {
```

Comment:
not an issue, just provide an alternative way.

```suggestion
    if (!userReport.userId && (userReport.countries?.length ?? 0) === 0) {
```